.travis.yml

```diff
@@ -22,12 +22,20 @@ env:
     - TESTS=group7
 
 before_install:
+  # Keycloak build requires custom Maven version. Let's grab the latest one
+  # https://github.com/keycloak/keycloak/blob/master/testsuite/integration-arquillian/pom.xml#L307
+  - wget https://archive.apache.org/dist/maven/maven-3/3.6.3/binaries/apache-maven-3.6.3-bin.zip
+  - unzip -qq apache-maven-3.6.3-bin.zip
+  - export M2_HOME=$PWD/apache-maven-3.6.3
+  - export PATH=$M2_HOME/bin:$PATH
```

Comment:
The answer is in my previous comment - https://github.com/keycloak/keycloak-quickstarts/pull/166#issuecomment-589527189

Making long story short - yes, we build Keycloak Server in Travis and Travis has different Maven version than is required by Keycloak.

Reply:
Don't we build Keycloak Server in Travis for the main repo? Or does this repo have a different Maven version from https://github.com/keycloak/keycloak?

Reply:
The only thing I found different was `dist: trusty` in this repo. It's a bit weird. If anyone has a clue why Travis picked up wrong Maven version, it would be great.

Reply:
Only the testsuite requires Maven 3.6 at the moment, so this is not needed as the QuickStarts doesn't need to build the testsuite, it can use -DskipTestsuite.

Reply:
Ok, let's give it a try.